moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -345,15 +370,43 @@ def trajectory_execution(
         self.__moveit_configs.trajectory_execution = {
             "moveit_manage_controllers": moveit_manage_controllers,
         }
-        self.__moveit_configs.trajectory_execution.update(
-            load_yaml(
-                self._package_path
-                / (
-                    file_path
-                    or self.__config_dir_path / f"{self.__robot_name}_controllers.yaml"
+
+        # Find the most likely controller params as needed
+        if file_path is None:
+            config_folder = self._package_path / self.__config_dir_path
+            controller_pattern = re.compile("^(.*)_controllers.yaml$")
+            possible_names = get_pattern_matches(config_folder, controller_pattern)
+            if not possible_names:
+                raise RuntimeError(
+                    "trajectory_execution: `Parameter file_path is undefined "
+                    f"and no matches for {config_folder}/*_controllers.yaml"
                 )
-            )
-        )
+            else:
+                chosen_name = None
+                if len(possible_names) == 1:
+                    chosen_name = possible_names[0]
+                else:
+                    # Try a couple other common names, in order of precedence
+                    for name in [self.__robot_name, "ros", "ros2"]:
```

Comment:
`ros/ros2_controllers.yaml` have to be loaded to ros2 controllers not moveit's trajectory execution manager, you renamed the file in https://github.com/ros-planning/moveit_resources/pull/127/files to moveit_controllers.yaml, what do you think about using that as a possible name?

```suggestion
                    for name in [self.__robot_name, "moveit"]:
```